<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTag</class>
 <widget class="QDialog" name="EditTag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier un tag</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="title">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modifier un tag</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="styleSheet">
        <string notr="true">color: red</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nomLabel">
         <property name="text">
          <string>Tag</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nomLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="changeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Changer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EditTag</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nomLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditTag</receiver>
   <slot>enableButton(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>EditTag</receiver>
   <slot>deleteTag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeButton</sender>
   <signal>clicked()</signal>
   <receiver>EditTag</receiver>
   <slot>changeTag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableButton(QString)</slot>
  <slot>deleteTag()</slot>
  <slot>changeTag()</slot>
 </slots>
</ui>
